README.md

```diff
@@ -8,15 +8,33 @@ Spinnaker ROS Driver
 ## Dependencies
 
 1. [ROS](http://wiki.ros.org/Installation/)
-1. The Spinnaker SDK, included under `spinnaker_driver`
+1. The Spinnaker SDK found at [FLIR](https://meta.box.lenovo.com/v/link/view/a1995795ffba47dbbe45771477319cc3).
+    - Navigate to the download directory, unzip the folder, navigate into the folder and run:
+    ```
+    ./install_spinnaker.sh
+    ```
+1. The AMRL shared library [amrl_shared_lib](https://github.com/ut-amrl/amrl_shared_lib) and AMRL configuration reader [config_reader](https://github.com/ut-amrl/config-reader). NOTE that both are installed during submodule initialization in the Build instructions by default.
+1. Install dependencies:
+    ```
+    sudo apt install liblua5.1-0-dev
+    ```
 
 ## Build
 
-1. Add the project directory to the `ROS_PACKAGE_PATH` environment variable.
+1. Navigate to the `spinnaker_ros_interface` directory.
+
+1. Initialize the submodules:
+    ```
+    git submodule update --init
```

Comment:
You can avoid having to do this by specify `--recurse-submodules` when cloning the repo.

Reply:
I think for a lot of people they forget to think about dealing with submodules and its not a bad idea to include this as a reminder.